docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+current master branch, it is possible that you will encounter errors.
+
+## Before you begin
+
+Before you can run the test network, you need to clone the `fabric-samples`
+repository and download the Fabric images. Make sure that that you have installed
+the [Prerequisites](../prereqs.html) and [Installed the Samples, Binaries and Docker Images](../prereqs.html).
+
+## Bring up the test network
+
+You can find the scripts to bring up the network in the `test-network` directory
+of the ``fabric-samples`` repository. Navigate to the test network directory by
+using the following command:
+```
+cd fabric-samples/test-network
+```
+
+In this directory, you can find an annotated script, ``network.sh``, that stands
+up a Fabric network using the Docker images on your local machine. You can run
+``network.sh -h`` to print the script help text:
+
+```
+network.sh <mode> [-c <channel name>] [-ca <use CAs>] [-t <timeout>] [-d <delay>] [-f <docker-compose-file>] [-s <dbtype>] [-l <language>] [-i <imagetag>] [-v]
+   <mode> - Bring up fabric network right away using ./network.sh up
+     - 'up' - bring up fabric orderer and peer nodes
+     - 'up createChannel' - bring up fabric network with one channel
+     - 'createChannel' - create and join a channel after the network is created
+     - 'deployCC' - Deploy a chainocode on the channel
+     - 'down' - clear the network with docker-compose down
+     - 'restart' - restart the network
+
+   Flags:
+   -c <channel name> - channel name to use (defaults to "mychannel")
+   -ca <use CAs> -  create Certificate Authorities to generate the crypto material
+   -r <max retry> - CLI times out after certain number of attempts (defaults to 5)
+   -d <delay> - delay duration in seconds (defaults to 3)
+   -f <docker-compose-file> - specify which docker-compose file use (defaults to docker-compose-cli.yaml)
+   -s <dbtype> - the database backend to use: goleveldb (default) or couchdb
+   -l <language> - the programming language of the chaincode to deploy: go (default), javascript, or java
+   -v <version>  - chaincode version. Must be a round number, 1, 2, 3, etc
+   -i <imagetag> - the tag to be used to launch the network (defaults to "latest")
+   -verbose - verbose mode
+ network.sh -h (print this message)
+
+Example use
+
+Taking all defaults:
+ network.sh up
+
+Using flags:
+ network.sh up createChannel -ca -c mychannel -s couchdb -i 1.4.0
+ network.sh createChannel -c channelName
+ network.sh deployCC -l node
+```
+
+From inside the `test-network` directory, you can bring up the network by issuing
+the following command. You will experience problems if you try to run the script
+from another directory:
+```
+./network.sh up
+```
+
+This command creates a Fabric network that consists of two peer nodes, one
+ordering node, and an instance of the Fabric tools container. If the command
+completes successfully, you script will print out the following logs of the nodes
```

Comment:
This seems misplaced. You're in effect asking people to run a test network when they have no idea what it contains.

I get why --- people shouldn't have to dig around for the script command, but I would at least link to the "components of the network section" somewhere near the beginning so that first time readers can jump down and find out what's in the network they're deploying.

Reply:
I am going to leave as is. People have the notion of a blockchain network before they know what it does. The components overview is the directly section below, so I do not think that there needs to be a link.